mod/wikirate/set/right/citation_count.rb

```diff
@@ -1,14 +1,37 @@
 format :html do
   view :titled do |args|
-    args.merge!(:slot_class=>( "no-citations" if card.format.render_core=="0" ))
+    args.merge! slot_class: ('no-citations' if no_citations?(args))
     super(args)
   end
 
   view :related_overview_modal_box do |args|
+    content_tag :div do
+      subformat(related_overview_card)
+        ._render_modal_link(args[:link_args])
+    end
+  end
+
+  def default_related_overview_modal_box_args args
+    link_text = subformat(card).render_titled(
+             args.merge(title: "Citations", hide: 'menu')
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.